src/modules/visitor-space/components/VisitorSpaceSearchPage/VisitorSpaceSearchPage.tsx

```diff
@@ -576,17 +590,18 @@ const VisitorSpaceSearchPage: NextPage = () => {
 									clearLabel={t(
 										'pages/bezoekersruimte/slug___wis-volledige-zoekopdracht'
 									)}
+									inputState={searchBarInputState}
```

Comment:
optional: might be cleaner to destructure the useState and reconstruct it here
so the other usages of searchBarInputState are more clear:

```
const [searchBarInput, setSearchBarInput] = useState<string>();

const searchValue = prepareSearchValue(searchBarInput);

setSearchBarInput(undefined);

inputState={[searchBarInput, setSearchBarInput]}
```